### PR TITLE
Don't split Quoted Printable characters during folding procedure

### PR DESCRIPTION
**Pull request by Gonzih**
I noticed that subjects sometimes are misformatted in my mail client due to improperly splitted quoted printable characters. Here is a small fix that should address this issue.

Thanks!

### CODE REVIEW

File: header.go [hunk 1]
```diff
@@ -40,6 +41,8 @@ func formatHeaderField(k, v string) string {
 		return s + "\r\n"
 	}
 
+	qpReg := regexp.MustCompile("(=[0-9A-Z]{2,2})+")
```
Comment (emersion) [inline header.go]: Can you move this outside the function, as a global variable?

**Comment by emersion** [comment]
Thanks!

**Comment by Gonzih** [comment]
Cool, thanks. Only problem I just noticed with this change is that it insertes extra whitespace before QP character. Can this whitespace be ommited? I can submit another PR in case if I can just drop whitespace if folding step is gonna happen on QP regexp and not a whitespace.

**Comment by emersion** [comment]
>Can this whitespace be ommited?

I don't think so, but let me know if you find a way to do that. The whitespace at the beginning of the line is used to mark folding.

**Comment by Gonzih** [comment]
What about then folding only on whitespaces or doing hard split in case 
its a one long word? This patch solves the issue, but still sometimes 
inserts whitespace before single or double quote for example.

**Comment by emersion** [comment]
Yeah, that would be better.

**Pull request merged**